### PR TITLE
Library: Update plugin-core to version 0.11.0

### PR DESCRIPTION
**Pull request by Zorin95670**

### CODE REVIEW

File: src/components/ModelizerModelView.vue [hunk 1]
```diff
@@ -75,6 +75,11 @@ async function drawComponents(plugin) {
   );
   // TODO: Remove this when plugin-core manage null as content.
   config.content = (!config.content || config.content === '') ? '{}' : config.content;
+  fileInputs.forEach((file) => {
```
Comment (k-y0u) [inline src/components/ModelizerModelView.vue:78]: Should not be done here. Better be in Project.js/readProjectFile.

File: src/composables/Project.js [hunk 2]
```diff
@@ -310,7 +310,7 @@ export async function writeProjectFile(projectId, file) {
   return new Promise((resolve, reject) => {
     fs.writeFile(
       `${projectId}/${file.path}`,
-      file.content,
+      file.content || '',
```
Comment (k-y0u) [inline src/composables/Project.js:313]: when do you encounter the case where content is null/undefined ?

**Review by k-y0u — CHANGES_REQUESTED** [review]

File: src/composables/events/PluginEvent.js [hunk 3]
```diff
@@ -19,10 +19,11 @@ const SelectEvent = new Subject();
 const EditEvent = new Subject();
 
 /**
- * Represent a rxjs Event object to emit and to receive events about plugins component deletion.
- * @typedef {Subject} DeleteEvent
+ * Represent a rxjs Event object to emit and to receive events when plugin have updated the
```
[the remark this answers is not in the document]
Comment (Zorin95670) [reply src/composables/events/PluginEvent.js:22]: done

File: src/composables/PluginManager.js [hunk 4]
```diff
@@ -64,9 +64,8 @@ export async function instantiatePlugin(pluginName) {
   const plugin = new plugins[pluginName]();
 
   plugin.init({
-    SelectEvent: PluginEvent.SelectEvent,
-    EditEvent: PluginEvent.EditEvent,
-    DeleteEvent: PluginEvent.DeleteEvent,
+    SelectEvent: PluginEvent.EditEvent,
+    UpdateEvent: PluginEvent.UpdateEvent,
```
Comment (justinehell) [inline src/composables/PluginManager.js:68]: why delete EditEvent ?

**Review by justinehell — CHANGES_REQUESTED** [review]

**Review by k-y0u — APPROVED** [review]

**Review by justinehell — DISMISSED** [review]

**Review by k-y0u — APPROVED** [review]

**Pull request merged**